DEVAI-213: Add Techdocs To Main Template (#48)
redhat-ai-dev/ai-lab-template: skeleton/template-card-techdocs/base/docs/index.md

# AI Software Template

> This AI Software Template can be used to create a new source code repository and new gitops deployment repository for a sed.edit.NAME application.

Reviewer: That's nice! We could use the other env vars too to provide more specific content (e.g APP_DESC or APP_SUMMARY). wdyt?

Author: At one point I was trying to incorporate both of those values into the doc but couldn't really find a good way to tie them in without looking out of place. If we wanted them to have their own section I could do that but worry about duplicating information since it is in the description as is

wdyt

Reviewer: Yeah I can see the point. I think after all about the content and what we should include it might needs a review from the docs team.

Author: +1 for that. I know I was loosely referencing some examples here for content: https://github.com/redhat-developer/red-hat-developer-hub-software-templates/blob/main/templates/create-backend-plugin/docs/index.md

> This AI Software Template can be used to create a new source code repository and new gitops deployment repository for a sed.edit.NAME application.

Reviewer 2: ```suggestion
This AI Software Template can be used to create a new source code repository and a new GitOps deployment repository for a sed.edit.NAME application.
```

Remove `application` when you make the switch to the APP_DISPLAY_NAME env

> As a user you will be able to enter various customizations for each section of the template, such as:

Reviewer 2: ```suggestion
As a user you will be able to customize each section of the template, such as:
```

### **Application Information**

- Application name

- Owner

- Model server of choice

### **Repository Information**

> - GitHub/GitLab Repository

Reviewer 2: ```suggestion
- GitHub/GitLab repository
```

Capitalization consistentcy

> - Desired repository owner

Reviewer 2: ```suggestion
- Repository owner
```

> - Desired name and branch for the repository

Reviewer 2: ```suggestion
- Name and branch for the repository
```

### **Deployment Information**

- Application deployment namespace

> - Desired image registry and image name

Reviewer 2: ```suggestion
- Image registry and image name
```
